devicetypes/smartthings/ezex-temp-humidity-sensor.src/i18n/messages.properties

```diff
@@ -0,0 +1,17 @@
+#  Copyright 2019 SmartThings
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy
+#  of the License at:
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+# Korean (ko)
+# Device Preferences
+'''Smart Temperature/Humidity Sensor(AC Type)'''.ko=스마트 온도/습도센서
```

Comment:
I think you'll need to update the space before the '(' here in the key, too, so that the translations work?